tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -156,15 +156,15 @@ exports[`test hug.js 1`] = `
   {__DEV__
     ? this.renderDevApp()
     : <div>
-      {routes.map(route => (
-        <MatchAsync
-          key={\`\${route.to}-async\`}
-          pattern={route.to}
-          exactly={route.to === \"/\"}
-          getComponent={routeES6Modules[route.value]}
-        />
-      ))}
-    </div>}
+        {routes.map(route => (
+          <MatchAsync
+            key={\`\${route.to}-async\`}
+            pattern={route.to}
+            exactly={route.to === \"/\"}
+            getComponent={routeES6Modules[route.value]}
+          />
+        ))}
+      </div>}
```

Comment:
I do want to call this out – currently, this outputs the closing `}` on the final line, which is indented. That might make it a bit harder to scan a JSX doc and see where a conditional ends. 

On the other hand, I've always found the dangling `}` annoying and ugly. 

On the other other hand, we probably want to be consistent about having closing punctuation that wraps indented content live on its own line (eg; `>` in an opening JSX element, `)` for a function call – though I haven't looked at the behavior of that one recently). 

I don't have an opinion here myself, but thought I should highlight it as potentially worth discussing.